package org.springapp.testtask.exception;

public class UserCreatedException extends Exception {
    public UserCreatedException(String message) {
        super(message);
    }
}
